content/docs/how-we-work/release-versioning.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Release Versioning
+weight: 14
+description: We give attention to release versioning as it is the practice of assigning unique identifiers or version numbers to software releases, indicating changes, updates, or improvements made to the software. These version numbers typically follow a specific format or convention, allowing developers, users, and other stakeholders to understand the significance of each release and track the evolution of the software over time.
+---
+
+# Release Versioning
+
+We give attention to release versioning as it is the practice of assigning unique identifiers or version numbers to software releases, indicating changes, updates, or improvements made to the software. These version numbers typically follow a specific format or convention, allowing developers, users, and other stakeholders to understand the significance of each release and track the evolution of the software over time.
```

Comment:
How do we do this? Do we create tags, changelog, release notes, or other assets for a release? Are there scripts available? What about manual steps?

Reply:
We create Releases and release notes as given [above](https://github.com/axelerant/engg-handbook/pull/163#discussion_r1861543697). We have not created tags however with baseline we would be doing that as well

Reply:
@Yemaneberhan-Lemma, let's update this to include more details. For now, let's mention Jira's releases feature since we use that.

Reply:
@hussainweb @Yemaneberhan-Lemma I also found another [page](https://axelerant.atlassian.net/wiki/spaces/AH/pages/4077158650/Efficient+Release+Planning+and+Best+Practices?focusedCommentId=4904124434) in confluence which talks about Release planning.

Ideally we need to have that here instead after reviewing and updating the content as needed

Reply:
@Kalaiselvan88, are you saying we should copy that content over here? Why not do that in this PR itself?

Reply:
@hussainweb I was assuming if @Yemaneberhan-Lemma can review and move it here. The content in that page was also available already and was not written by me. But anyway I will update this PR to include that as well

Reply:
Will raise another PR for further changes. Now merging this